keyboards/ungodly/launch_pad/rules.mk

```diff
@@ -2,7 +2,7 @@
 MCU = atmega32u4
 
 # Bootloader selection
-BOOTLOADER = atmel-dfu
+BOOTLOADER = qmk-dfu
```

Comment:
If you really are using qmk-dfu, you should have the corresponding defines in the config.h:
https://docs.qmk.fm/#/flashing?id=qmk-dfu

Reply:
I'll roll it back. Only using it for production hex.